.npmignore

```diff
@@ -4,4 +4,6 @@
 /tests/
 lcov-*
 xunit*
-components/
\ No newline at end of file
+components/
+src/
```

Comment:
can we keep this in the package too? Just in case someone is full ES6 wants to use the src version